src/main/java/seedu/address/storage/JsonSerializableTbmManager.java

```diff
@@ -61,11 +61,10 @@ public TbmManager toModelType() throws IllegalValueException {
             }
             tbmManager.addClient(client);
         }
-        for (JsonAdaptedNote note: notes) {
+        for (JsonAdaptedNote note: countryNotes) {
```

Comment:
Renaming `note` to `countryNote` might be clearer

Reply:
renamed to `jsonAdaptedCountryNote`